src/files/source/tree.php

```diff
@@ -238,33 +238,72 @@
         'role:sale.master',
     ],
     // STOCK MODULE
-    'role:stock.user' => [
-        'stock.read', 'part.read', 'move.read', 'model.read', 'move.get-directions', 'order.read',
+    'role:part.user' => [
+        'part.read',
     ],
-    'role:stock.admin' => [
-        'role:stock.user',
-        'move.create', 'move.update', 'move.delete',
+    'role:part.manager' => [
+        'role:part.user',
+        'part.create', 'part.update', 'part.delete',
     ],
-    'role:stock.manager' => [
-        'role:stock.user',
+    'role:part.master' => [
         'part.read-all-hierarchy',
-        'part.create', 'part.update', 'part.delete',
+        'role:part.manager',
+    ],
+    'role:move.user' => [
+        'move.read',
+        'move.get-directions',
```

Comment:
Why do you think a regular user would need this?
As far as I see, it's being used in edit forms.